examples/nodejs-simple/README.md

```diff
@@ -1,4 +1,66 @@
-### Instructions
-`npm install` - install dependencies
-
-`npm start` - start the self-running example
+# Simple node.js Example
+
+This is a very simple "server" that doesn't do much other than show how the SDK works in node.
+
+It will
+- Setup the Agones SDK
+- Call `sdk.ready()` to register that it is ready with Agones.
+- Call `sdk.health()` a send a health check message.
+- Call `sdk.shutdown()` to shut the server down.
+
+## Running in a cluster
+
+These instructions allow you to run the example code without building a container, but they require you to
+have a cluster with Agones installed. If you don't yet have a cluster, check out
+[these instructions](https://agones.dev/site/docs/installation/).
+
+To run the example server:
+```
+kubectl create -f https://raw.githubusercontent.com/googleforgames/agones/master/examples/nodejs-simple/gameserver.yaml
```

Comment:
Same thing as others. use `release-branch` in some way?

Reply:
Do you think it's overkill to have a tutorial for each _sdk_-simple example like I did for C++?

Reply:
I don't think any user is going to complain about more valuable documentation 🤷‍♂️ WDYT?

Reply:
If we standardise on how examples are built - we could maybe have a common shortcode(s) with variables that repeat the common parts across each tutorial to make consistency and authoring easier? Just an idea.

Reply:
I was thinking about that too. The C++ and rust examples are basically identical, and the node one is similar (but not exactly the same). I haven't looked at the others yet to know if we could template this out. Or even if we could do it for some of them that might be nice. 

It isn't too difficult to write the tutorial (once you have the code working -- output not flushing took many hours to track down) so I'll add one here following the same pattern as with the c++ one.

Reply:
I really like what you did for the C++ sdk - I think that is super useful for users 👍

Reply:
Should I want for the tutorial on this? I'm otherwise LGTM on this PR.

Reply:
Yes, let me add it in.

Reply:
Updated.